bevy_lint/action.yml

```diff
@@ -43,7 +65,48 @@ runs:
         toolchain: ${{ steps.toolchain.outputs.channel }}
         components: ${{ steps.toolchain.outputs.components }}
 
+    - name: Create cache key
+      id: cache-key
+      if: ${{ inputs.cache == 'true' }}
+      shell: bash
+      run: echo "key=bevy_lint-${RUNNER_OS}-${ACTION_REF}" >> "${GITHUB_OUTPUT}"
```

Comment:
This key makes it so caches for different operating systems and linter versions won't be shared. For example, Windows and MacOS will have separate caches, and `bevy_lint` v0.3.0 and `bevy_lint` `main` will have separate caches.